Cargo.toml

```diff
@@ -86,6 +86,7 @@ once_cell = "1.19"
 [build-dependencies]
 fslock = "0.2"
 which = "5"
+home = "0"
```

Comment:
Wow, that's a huge dependency, good that it's build only

Reply:
I ran cargo update to fix the lockfile -- no more extra deps.